stdlib/tests/collections/smt.rs

```diff
@@ -0,0 +1,98 @@
+use crate::build_test;
+use test_utils::{
+    get_remaining_key, proptest::prelude::*, seeded_word, EmptySubtreeRoots, Felt, IntoBytes,
+    MerkleStore, NodeIndex, Rpo256, StarkField, Word, WORD_SIZE, ZERO,
+};
+
+proptest! {
+    #[test]
+    fn arbitrary_smtget_opens_correctly(
+        ref seed in any::<[u8; 32]>()
+    ) {
+        fn assert_case(
+            key: Word,
+            value: Word,
+            root: Word,
+            store: MerkleStore,
+            advice_map: &[([u8; 32], Vec<Felt>)],
+        ) {
```

Comment:
Is there a reason to keep this as an inner function? It would make reading the test a bit easier if this was separate, I think.

Reply:
The only reason is the convention I use: when something is used only for a single test case, it should be declared inside its block (consts, fns, modules, etc)

I moved outside and named it `assert_case_arbitrary_smtget_opens_correctly`; let me know which variant you prefer

Reply:
I think a separate function is better (especially in this case as we can use it in multiple tests).

Reply:
Sounds good! Removed